docs/ref/django-admin.txt

```diff
@@ -226,6 +226,23 @@ program manually.
 
 Specifies the database onto which to open a shell. Defaults to ``default``.
 
+.. django-admin-option:: -- <args>
+
+.. versionadded:: 3.1
+
+Any arguments following a ``--`` divider will be passed on to the underlying
+program. For example, with MySQL/MariaDB, you can use the ``mysql`` command's
+``-e`` flag like so:
+
+.. console::
+
+    $ python manage.py dbshell -- -e "SELECT NOW()"
+    +---------------------+
+    | NOW()               |
+    +---------------------+
+    | 2020-02-29 16:28:49 |
+    +---------------------+
+
```

Comment:
I just wonder if we can help folks a little more here, showing them the basic option for each supported DB? 

(`-c`, `--cmd`, `...😬...`) 

Danger is it gets too verbose. Would certainly need a "See your DB's ..."

Reply:
I've gone halfway with a Postgres example first, given it's like 90% of the community.